standalone-metastore/src/test/java/org/apache/hadoop/hive/metastore/TestAggregateStatsCache.java

```diff
@@ -46,15 +49,15 @@
   static int NUM_COLS = 5;
   static int MAX_CACHE_NODES = 10;
   static int MAX_PARTITIONS_PER_CACHE_NODE = 10;
-  static String TIME_TO_LIVE = "20s";
-  static String MAX_WRITER_WAIT = "1s";
-  static String MAX_READER_WAIT = "1s";
-  static float FALSE_POSITIVE_PROBABILITY = (float) 0.01;
-  static float MAX_VARIANCE = (float) 0.5;
+  static long TIME_TO_LIVE = 2;
```

Comment:
Did you mean to change this from 20 to 2?

Reply:
Yes, unit tests should run in under 5 seconds whenever possible.